paper/basic_training.tex

```diff
@@ -106,88 +101,79 @@ \section{Introduction}
 With MD methods, the equation of motion is numerically integrated and a dynamical trajectory of the system is generated. 
 MD simulations can be used for investigating structural, dynamic, and thermodynamic properties of the system.
 With MC methods, probabilistic rules are used to generate a new configuration from the present configuration and this process is repeated to generate a sequence of states that can be used to calculate structural and thermodynamic properties but not dynamical properties; indeed, MC simulations lack any concept of time. 
-Thus, the ``dynamics'' produced by an MC method is not the temporal dynamics of the system, in the sense that it may or may not follow the physical time evolution of the system. 
-However, MC can be used to study dynamical behavior in model systems or in systems where MD simulations cannot be carried out due to computational cost, with a suitable mapping between MC step-size and a physical unit of time. 
+Thus, the ``dynamics'' produced by an MC method are not the temporal dynamics of the system. 
+%However, MC can be used to study dynamical behavior in model systems or in systems where MD simulations cannot be carried out due to computational cost, with a suitable mapping between MC step-size and a physical unit of time.  %DLM: Is this ever true? Suggest removing.
```

Comment:
I think this is referring to something like kinetic MC, which can examine complex dynamical systems that cannot be solved analytically.  I agree with removing, though, as it will just confuse things.